testdata/cldr-testData/units/_readme.txt

```diff
@@ -0,0 +1,6 @@
+# Test data for units: conversions and preferences
```

Comment:
FMI: What's with the leading underscore?

Reply:
I took a look around, couldn't find an answer, just an observation:

- All readme.txt files in the common/testData tree have a leading underscore.
- No other readme files in CLDR have a leading underscore.

Reply:
Hmm, okay. Seems a bit janky, but there's probably a reason (e.g. visual file ordering in Windows or something).